data/rulesets.js

```diff
@@ -256,7 +262,8 @@ exports.BattleFormats = {
 		},
 	},
 	hoennpokedex: {
-		effectType: 'Rule',
+		effectType: 'Banlist',
```

Comment:
Adding support for changing rulesets (only in <code>/tour banlist</code> for now) involves using <code>Tools.getFormat()</code> and checking the result's <code>effectType</code>. Since supplementary banlists only affect team validation and not in-battle rules, this commit allows it to limit changes to those that have an effect by checking <code>effectType === 'Banlist'</code> instead of checking for all properties that affect validation (<code>ruleset</code>, <code>banlist</code>, <code>onValidateTeam</code>, <code>onValidateSet</code>, etc.). So <code>/tour banlist same type clause, !cancel mod</code> would only show Same Type Clause being added, since Cancel Mod cannot actually be removed.